ext/js/dictionary/dictionary-data-util.js

```diff
@@ -351,7 +351,7 @@ export class DictionaryDataUtil {
      * @param {T[]} array2
      * @returns {boolean}
      */
-    static _areArraysEqual(array1, array2) {
```

Comment:
Why did you remove the underscore prefix?

Reply:
I used it outside the class so it's no longer "private".

Reply:
Is it in this pr? I see no references outside of the class:
![Uploading image.png…]()

Reply:
It's in `translator.js`, though we'll probably change that back and use a different function (see above)